circleci/requirements.txt

```diff
@@ -0,0 +1,9 @@
+wheel
```

Comment:
I've noticed we often `pip install wheel` before installing modules of the requirements.txt file. Seems it was done for performance purpose. As wheel is mainly used it's often in the modules dependencies, so I put it first just in case. Not sure there is a real gain though

Reply:
Could we pin the `wheel` version used, for safety?